src/main/cpp/subsystems/Winch.cpp

```diff
@@ -21,7 +39,8 @@ void Winch::Periodic()
 
 }
 
-void Winch::WinchHook(double percentoutput)
+void Winch::WinchHook(double position)
```

Comment:
We should add a new interface. We probably want a simple test that doesn't involve the control system.